testing-guidance/workflow-tests/animation/animation-editor-workflows.md

```diff
@@ -0,0 +1,115 @@
+| **Launch and configure the Animation Editor**                                     | <ol><li>Open the O3DE Editor.</li><li>Open the Animation Editor (using varying entry points):</li><ul><li>O3DE Toolbar → Tools → Animation Editor</li><li>Actor Component's Actor asset property field.</li></ul><li>Resize the Animation Editor to desired size.</li><li>Dock/Undock editor to desired position.</li><li>Select the desired Layout.</li><li>Move Animation Editor panels around within the Animation Editor</li><li>Open and close Separator Widgets on various panels.<ul><li>Joint Outliner</li><li>Motion Sets</li><li>Motion Events</li><li>Actor Manager</li></ul></li></ol>                                                                                                                                                                        | <ul><li>Animation Editor launches</li><li>Animation Editor Resizes.</li><li>Animation Editor docks as expected.</li><li>Layouts will quickly switch between different View configurations.</li><li>Separator widgets can be collapsed and expanded while the parent layout resizes accordingly. [o3de/o3de#10958](https://github.com/o3de/o3de/issues/10958), [o3de/o3de#10960](https://github.com/o3de/o3de/issues/10960) <ul><li>![](images/separator_widget.gif) </li></ul></li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
update: @FuzzyCarterAWS please can you remove the section about the 'Joint Outliner' - it has been removed and replaced with the 'Skeleton Outliner' (see https://github.com/o3de/o3de/issues/11311)

Reply:
Removed deprecated reference.